pkg/vorteil/launch.go

```diff
@@ -105,10 +105,13 @@ func (p *program) waitForApp(cmd *exec.Cmd) {
 	err := cmd.Wait()
 	if err != nil {
 		logDebug("error while waiting: %s", err.Error())
-	} else {
-		logDebug("process %d finished with %s", cmd.Process.Pid, cmd.ProcessState.String())
 	}
 
+	// Returns exit status
+	logDebug("process %d finished with %s", cmd.Process.Pid, cmd.ProcessState.String())
+
+	p.isDone = true
```

Comment:
instead of isDone, we should use the return value which is an int. We can use negative values for hasn't started, running etc. and 0 and other positives for return values. With that you can check if you need to send a signal at all, e.g. they haven't started or have finished.